pages/blog/_slug.vue

```diff
@@ -36,3 +36,15 @@ export default {
   },
 };
 </script>
+
+<style>
+.content {
+  /* ... */
+  display: grid;
+  grid-template-columns: 1fr min(100ch, 100%) 1fr;
```

Comment:
Maybe add `padding-inline: 16px` or something here? Just so it never is flush with the sides of the screen.

Reply:
added padding to the parent container class!